proto/storage/group.proto

```diff
@@ -26,6 +27,7 @@ message Group {
 message GroupProperties {
```

Comment:
Add a hint about modifying `GroupProperties` may require changes to `DeleteGroupRequest`.